docker/tinode/Dockerfile

```diff
@@ -102,6 +102,7 @@ RUN tar -xzf tinode-$TARGET_DB.linux-amd64.tar.gz \
 	&& rm tinode-$TARGET_DB.linux-amd64.tar.gz
 
 # Copy config template to the container.
+COPY init-db .
```

Comment:
I believe this line copies init-db from your host docker directory to container. You should already have it in your container after downloading and unpacking `tinode-$TARGET_DB.linux-amd64.tar.gz`.

Reply:
This line is not necessary. Removed.